virtualbox/resource_vm.go

```diff
@@ -80,6 +80,13 @@ func resourceVM() *schema.Resource {
 				Default:  "512mib",
 			},
 
+			"firmware": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "Firmware of the vm, can either be BIOS or EFI",
```

Comment:
What happens if the user provides the values in lower case, like `bios` or `efi`?